bridge/client/app/_components/ktb-sequence-list/ktb-sequence-list.component.html

```diff
@@ -3,44 +3,50 @@ <h2>Sequences</h2>
   <ng-container dtColumnDef="time" dtColumnAlign="text" dtColumnProportion="2">
     <dt-header-cell *dtHeaderCellDef>Started at</dt-header-cell>
     <dt-cell *dtCellDef="let row">
-      <ng-container *ngIf="isRemediation(row), else trace">
+      <ng-container *ngIf="isRemediation(row) as sequence, else trace">
```

Comment:
I would implement the `isRemediation` method in the sequence model, which makes it reusable.